app/pom.xml

```diff
@@ -0,0 +1,259 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+    <artifactId>application</artifactId>
+    <name>The Aplication that runs Vaadin Flow and in OSGi</name>
+
+    <packaging>pom</packaging>
+    <parent>
+        <groupId>com.example</groupId>
+        <artifactId>project-base-osgi-parent</artifactId>
+        <version>1.0-SNAPSHOT</version>
+    </parent>
+
+    <properties>
+        <bnd.version>5.2.0</bnd.version>
+        <extra.bundles.dir>${project.build.directory}/extra-bundles</extra.bundles.dir>        
+    </properties>
+
+    <dependencies>
+        <dependency>
+            <groupId>com.example</groupId>
+            <artifactId>project-base-osgi</artifactId>
+            <version>${project.version}</version>
+        </dependency>
+        
+        <dependency>
+            <groupId>com.vaadin</groupId>
+            <artifactId>vaadin-button-flow</artifactId>
+        </dependency>
```

Comment:
A dumb question - why is this here ?

Reply:
The question is good.
It's a leftover from the first attempt to make the starter work without 19 Platform.
Now it's not needed anymore.